universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -67,6 +68,12 @@ public ApplicantProgramReviewController(
         .thenApplyAsync(
             (roApplicantProgramService) -> {
               ImmutableList<AnswerData> summaryData = roApplicantProgramService.getSummaryData();
+              int allBlockCount = roApplicantProgramService.getAllBlocks().size();
```

Comment:
nit: I passed this number around elsewhere in the views/controllers and called it `totalBlockCount`. Could update this one to match since it represents the same number.

Reply:
I used total in other places. idk why i did this 🤷